ci-operator/step-registry/cucushift/installer/check/control-plane-machinesets/cucushift-installer-check-control-plane-machinesets-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+
+# control-plane machinesets
+stderr=$(mktemp)
+stdout=$(mktemp)
+oc get controlplanemachinesets -n openshift-machine-api 1>${stdout} 2>${stderr} || true
```

Comment:
How about `oc get controlplanemachinesets cluster -n openshift-machine-api --no-headers` for cleaner output?

Reply:
@jianlinliu If the check passed, the stdout will finally be print out. I think with headers in stdout would be better, as there are multiple columns. @yunjiang29 WDYT?

Reply:
Actually, both are acceptable to me,  but no header is stricter for program checking.